src/clusterfuzz/_internal/bot/tasks/utasks/variant_task.py

```diff
@@ -74,16 +74,18 @@ def utask_preprocess(testcase_id, job_type, uworker_env):
   setup_input = setup.preprocess_setup_testcase(testcase)
   testcase_upload_metadata = data_types.TestcaseUploadMetadata.query(
       data_types.TestcaseUploadMetadata.testcase_id == int(testcase_id)).get()
-  return uworker_msg_pb2.Input(
+  uworker_input = uworker_msg_pb2.Input(
       job_type=job_type,
       original_job_type=original_job_type,
       testcase=uworker_io.entity_to_protobuf(testcase),
-      testcase_upload_metadata=uworker_io.entity_to_protobuf(
-          testcase_upload_metadata),
+      testcase_upload_metadata=),
       uworker_env=uworker_env,
       testcase_id=testcase_id,
       setup_input=setup_input,
   )
+  if testcase_upload_metadata:
+    uworker_input.testcase_upload_metadata = uworker_io.entity_to_protobuf(
```

Comment:
I would have expected a `CopyFrom()` to be necessary here.

Reply:
You are correct: https://github.com/google/clusterfuzz/blob/master/src/clusterfuzz/_internal/bot/tasks/utasks/variant_task.py#L86